apps/web/src/app/api/projects/[projectId]/documents/[documentUuid]/evaluatedLogs/route.ts

```diff
@@ -0,0 +1,77 @@
+import { Workspace } from '@latitude-data/core/browser'
+import { computeDocumentLogsQuery } from '@latitude-data/core/services/documentLogs/computeDocumentLogs'
+import { authHandler } from '$/middlewares/authHandler'
+import { errorHandler } from '$/middlewares/errorHandler'
+import { NextRequest, NextResponse } from 'next/server'
+import { parseApiDocumentLogParams } from '@latitude-data/core/services/documentLogs/logsFilterUtils/parseApiLogFilterParams'
+import { serializeEvaluatedDocumentLog } from '@latitude-data/core/services/evaluationsV2/llm/serializeEvaluatedDocumentLog'
+
+export const GET = errorHandler(
+  authHandler(
+    async (
+      req: NextRequest,
+      {
+        params,
+        workspace,
+      }: {
+        params: {
+          projectId: string
+          documentUuid: string
+        }
+        workspace: Workspace
+      },
+    ) => {
+      const { documentUuid } = params
+      const searchParams = req.nextUrl.searchParams
+      const queryParams = parseApiDocumentLogParams({ searchParams })
+
+      if (+queryParams.pageSize > 1) {
+        return NextResponse.json(
+          {
+            message:
+              'At the moment we only support pageSize=1 for evaluated logs',
+          },
+          { status: 422 },
+        )
+      }
```

Comment:
you can just throw an unprocessable entity rather than cooking yourself the error response